VERSIONING.md

```diff
@@ -0,0 +1,9 @@
+Versioning policy declaration
+=============================
+
+Starting on v2.0.0, the version numbers for this library will follow [Semantic Versioning](https://semver.org/).
+
+This means that API breaks should only happen between major version changes.
+If an ABI break is required, it should be done between minor versions and be clearly stated on the release notes.
```

Comment:
I'm not sure whether we need these

Reply:
I prefer to be explicit, especially regarding ABI changes, as semver does not seem to be clear.